File: vue/src/store/entities/banner.ts
import Entity from './entity'

export default class Banner extends Entity<number>{
    showBeginDate:Date;
    showEndDate:Date;
    bannerOrder: number;
    linkType: number;
    bannerType: number;
    bannerImg: string

    constructor(){
        super();
        this.bannerOrder = 0;
        this.showBeginDate = new Date();
        this.showEndDate = new Date();
        this.linkType = 0;
        this.bannerType = 0;
        this.bannerImg = "";
    }
}

File: aspnet-core/src/MyABP.Core/Operation/Banner.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MyABP.Operation
{
    public class Banner: AuditedEntity<int>
    {
        public string BannerImg { set; get; }

        public string BannerLink { set; get; }

        public int BannerOrder { set; get; }

        public int Status { set; get; }

        public int Type { set; get; }

        public int LinkType { set; get; }

        public string Theme { set; get; }

        public DateTime ShowBeginDate { set; get; }

        public DateTime ShowEndDate { set; get; }


    }
}
